index.html

```diff
@@ -68,7 +68,493 @@ <h3>I am a Software Developer</h3>
+                                    <a href="#">
+                                        <button type="button"  class="professional__button-select">Boostrap</button>
+                                    </a>
+                                </li>
+                                <li>
+                                    <a href="#">
+                                        <button type="button"  class="professional__button-select">Ruby</button>
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="professional__see-project">
+                        See Project
+                    </div>
+                </div>
+
+            </section>
+
+            <section class="aboutme">
+                <div class="aboutme__container">
+                    <div class="aboutme__title">
+                        <h6>About me</h6>
+                    </div>
+                    <div class="aboutme__description">
+                        <p>
+                            Hello I’m a software developer! I can help <br> 
+                            you build a product , feature or website <br> 
+                            Look through some of my work and <br>
+                            experience! If you like what you see and <br> 
+                            have a project you need coded, don’t <br> 
+                            hestiate to contact me.
+                        </p>
+                    </div>
+                    <div class="aboutme__button">
+                        <a href="#" class="aboutme__link">
+                           Get  My Resume
+                        </a>
+                    </div>
+                </div>
+                <div class="aboutme__background-image">
+
+                </div>
+                <div class="boutme__separator">
+                    <div class="aboutme__separator-line">
+                    
+                    </div>
+                </div>
+                
+            </section>
+
+            <section>
+                <div class="aboutme__details">
+                    <div class="grid-container">
+                        <div class="item1">
+                            <img src="./images/kite_shape.png" alt="no internet" height="25px">
+                        </div>
+                        <div class="item2">Languages</div>
+                    </div>
+                    <ul>
+                        <li>
+                            <h3>Java Script</h3>
+                        </li>
+                        <li>
+                            <h3>Ruby</h3>
+                        </li>
+                        <li>
+                            <h3>HTML</h3>
+                        </li>
+                        <li>
+                            <h3>CSS</h3>
+                        </li>
+                    </ul>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, I would be better to align your listed items in the center to replicate the Figma design 👍 

As these elements are aligned by ```<ul>```, I think you may directly select this area in CSS and try to use ```display: flex;``` with ```flex-direction: column``` and also you add ```align-items: center;``` and this may help to replicate the Figma design 👍 👍 

~ Keep it up 😎